dbt_subprojects/dex/models/_projects/sushiswap/apechain/sushiswap_agg_apechain_trades.sql

```diff
@@ -0,0 +1,17 @@
+{% set chain = 'apechain' %}
+
+{{ config(
+  schema = 'sushiswap_' ~ chain,
```

Comment:
```suggestion
  schema = 'sushiswap_' + chain,
```

Reply:
side note: model config is different than jinja within a model. a bit annoying, but need a `+` here to properly apply

due to this hidden bug, your tables are actually building as the file name in directory rather than schema/alias applied here. you can tell by CI table names in the logs (contains `_agg_` in it, while you don't in model configs)

Reply:
also, plz look at existing sushiswap dex models and ensure you aren't clashing with naming standards there. maybe we should do `aggregator_trades` in alias for all of these?